shared/bundle_analysis/parser.py

```diff
@@ -62,6 +62,24 @@ def parse(self, path: str) -> int:
             for event in ijson.parse(f):
                 self._parse_event(event)
 
+            # Delete old session/asset/chunk/module with the same bundle name if applicable
+            old_session = (
+                self.db_session.query(Session)
+                .filter(
+                    Session.bundle == self.session.bundle, Session.id != self.session.id
+                )
+                .one_or_none()
+            )
+            if old_session:
+                for model in [Asset, Chunk, Module]:
```

Comment:
With all the cascading you set before, would this work if you only deleted the session?

Reply:
I would need to do a  `cascade="all, delete"` for Assets/Chunks/Modules as a relationship on the Sessions table, then when I delete the session it should delete the related ACM objects, then the other many-to-many table objects.
But for some reason add the `relationship` line in the Session model doesn't work, I've spent a decent amount of time on it before giving up :(